metadata/src/main/java/edu/unc/lib/dl/validation/MODSValidator.java

```diff
@@ -0,0 +1,96 @@
+package edu.unc.lib.dl.validation;
+
+import static edu.unc.lib.dl.xml.JDOMNamespaceUtil.SCHEMATRON_VALIDATION_REPORT_NS;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.stream.Collectors;
+
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+
+import org.jdom2.Document;
+import org.jdom2.output.XMLOutputter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
+
+import edu.unc.lib.dl.schematron.SchematronValidator;
+
+/**
+ * Asserts that MODS complies with the XSD and controlled vocabularies,
+ * as per the configured schematron validator.
+ *
+ * @author bbpennel
+ * @author harring
+ *
+ */
+public class MODSValidator {
+    private static final Logger log = LoggerFactory.getLogger(MODSValidator.class);
+
+    private SchematronValidator schematronValidator = null;
```

Comment:
is there a reason why these are set explicitly to null?

Reply:
nope :)  probably copy paste